gnucash/report/trep-engine.scm

```diff
@@ -1226,6 +1228,20 @@ be excluded from periodic reporting.")
                                   "number-cell"
                                   (xaccSplitGetAmount split)))))
 
+               (add-if (column-uses? 'association)
+                       (vector ""
+                               (lambda (split transaction-row?)
+                                 (let ((url (xaccTransGetAssociation
+                                             (xaccSplitGetParent split))))
+                                   (and (not (string-null? url))
+                                        (gnc:make-html-table-cell/markup
+                                         "text-cell"
+                                         (if opt-use-links?
+                                             (gnc:html-transaction-association-anchor
+                                              (xaccSplitGetParent split)
+                                              (_ "📎"))
+                                             (_ "📎"))))))))
+
```

Comment:
Did you mean to display the same character for both legs of the if? That aside, does it make sense to put the emoji-set characters in the Scheme code? Aside from it being better to have only one place where the codepoints are set so that it's easy to change them if we decide to, you can't send them like that to the html code, they need to be the corresponding html enitities. If they're going to be translated here they should be translated in the register too, and they need a translator comment explaining what they're for.

Reply:
Yes, I have now changed them to be the same as the register column title.